src/App.js

```diff
@@ -24,9 +24,6 @@ export default function App() {
         </nav>
 
         <Switch>
-          <Route path="/about">
-            <About />
-          </Route>
           <Route path="/favorites">
             <Favorites />
```

Comment:
<Route path='/favorites' component={Favorites} />, will make your life easier in the future. Remind you to show why in the coaching call

Reply:
I don't understand what you mean. I will remind you during the coaching call.